x-pack/legacy/plugins/spaces/index.ts

```diff
@@ -81,18 +81,14 @@ export const spaces = (kibana: Record<string, any>) =>
           serverBasePath: server.config().get('server.basePath'),
         };
       },
-      async replaceInjectedVars(
-        vars: Record<string, any>,
-        request: Legacy.Request,
-        server: Record<string, any>
-      ) {
-        const spacesPlugin = server.newPlatform.setup.plugins.spaces as SpacesPluginSetup;
+      async replaceInjectedVars(vars: Record<string, any>, request: KibanaRequest, server: Server) {
```

Comment:
Did something change with this PR so the legacy not-a-platform's `replaceInjectedVars` is now being called with a `KibanaRequest` instead of the `Legacy.Request`?

Reply:
Yes, but I did revert it to accept either type of request.